exporter/prometheusexporter/prometheus.go

```diff
@@ -58,6 +60,7 @@ func newPrometheusExporter(config *Config, set component.ExporterCreateSettings)
 			registry,
 			promhttp.HandlerOpts{
 				ErrorHandling: promhttp.ContinueOnError,
+				ErrorLog:      log.New(os.Stderr, "Prometheus Exporter Check Error ", 0),
```

Comment:
Please extend zap to implement that interface by creating our own type that extends that interface.

```golang
type foo zap.SugarLogger

func (*foo) Println(v ...interface{}) {
  foo.Error(v...)
}
```

Reply:
Can this be done in a followup PR @bogdandrutu?